app/views/admin/users/_member_index.html.erb

```diff
@@ -14,9 +14,28 @@
         </tr>
       </thead>
       <tbody class="crayons-card">
-        <% @users.each do |user| %>
+        <% @users.includes([:organizations]).each do |user| %>
         <tr>
-          <td><%= link_to user.name, admin_user_path(user) %></td>
+          <td>
+          <div class="flex s:items-start flex-col s:flex-row">
+            <a href="<%= admin_user_path(user.id) %>">
+              <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+            </a>
+            <div class="pl-3 flex-1"><%= user.name %>
+              <p class="fs-xs color-base-60">@<%= user.username %>&nbsp;<span class="fs-xs c-indicator" aria-disabled="true"><%= user.current_roles %></span></p>
+            </div>
+          </div>
+          </td>
+          <td><%= user.current_status %></td>
```

Comment:
I'm guessing for the coloured indicator next to this text we are going to need to do something like we have in `app/views/admin/users/show/profile/_status.html.erb` (a big else/if)

Unless we can add a ruby helper that would surface a `status_color` or similar, based on the user's data?

Reply:
Correct, we'll need a big helper! I'll be tackling that in a separate PR, as this PR just gets the bare-bones, necessary data in the columns as a first step. If you'd prefer, I can instead implement it in this PR, though.

Reply:
No, sounds good - let's keep it separate!

Reply:
Decided to circle back around to this and add it, as it was quicker than anticipated and since this PR is still actively in review!